students/piechowski_michal/lesson_12_debugging/mad_libs_text.txt

```diff
@@ -0,0 +1,2 @@
+The ADJECTIVE panda walked to the NOUN and then VERB 
```

Comment:
ten plik chyba nie jest potrzebny

Reply:
Potrzebny nie jest, ale poniewaz zadanie z verbose_output jest kopia zadania z lekcji 11, ktora wykorzystywala ten plik i obecne zadanie nadal sluzy do przeszukiwania plikow tekstowych, to dodalem tez ten plik, zeby dalo sie je testowac.